aries_cloudagent/wallet/routes.py

```diff
@@ -188,6 +188,12 @@ class DIDCreateSchema(OpenAPISchema):
         description="To define a key type for a did:key",
     )
 
+    seed = fields.Str(
+        required=False,
+        description="Optional seed to use for DID",
```

Comment:
```suggestion
        description="Optional seed to use for DID. Must be enabled in configuration before use.",
```

Reply:
Resolved

Reply:
I don't think we should advertise the `seed` unless the startup option is set, so if we can't make it conditional in the schema we should add a separate endpoint that we can enable/disable based on the `--wallet-allow-insecure-seed` parameter.
@swcurran  @andrewwhitehead  thoughts?

Reply:
We are returning an error if a seed is passed without the startup parameter. So I believe people will not get confused with the schema.

Reply:
@ianco I don't think we can add conditional endpoints or parameters as then we'd have to look at what options were passed when generating the openapi definition

Reply:
What is the reason we want to hide this? There's a lot of properties that are available based on startup parameters (using public dids for invitations requires both the `--public-invites` and a ledger to be configured to work), but are always available in the swagger ui. If not configured, an error will be thrown when you try to use it.

Sorry for pushing you in the wrong direction @DaevMithran.

Reply:
I think we want to discourage this (use of seed), unless there is a specific use case for it, in which case the implementer can figure it out.  If we expose this as a top-level parameter then I suspect everyone will just use it, in which case the extra startup parameter will just become one of the de-facto parameters that everyone uses.

Two separate endpoints seems like a good compromise to me, similar to the discussion we had around the multi-tenant bearer token endpoint.

However if I'm out-voted I'll go along with whatever is the consensus decision.  @swcurran  or @andrewwhitehead   can weigh in ...

Reply:
I guess the other option would be to require encryption of the seed when it's passed it, but that's another can of worms :)

Reply:
Ok let's keep it out of the docs then. I still think throwing an error would be nice, as this doesn't leave users thinking they've used the secret ACA-Py feature while it's actually not enabled and you don't have the seed anymore